src/sas/sascalc/dataloader/manipulations.py

```diff
@@ -602,7 +604,9 @@ def __call__(self, data2D, ismask=False):
         # Get data W/ finite values
         data = data2D.data[np.isfinite(data2D.data)]
         q_data = data2D.q_data[np.isfinite(data2D.data)]
-        err_data = data2D.err_data[np.isfinite(data2D.data)]
+        err_data = None
```

Comment:
I'm seeing the following error on many of the slicers and think `err_data` needs to be initialized for all manipulations.

> ERROR : sas.qtgui.Perspectives.Fitting.FittingWidget (FittingWidget.py:175) :: Traceback (most recent call last):
  File "sas\qtgui\Plotting\Plotter2D.py", line 380, in onAnnulusView
  File "sas\qtgui\Plotting\Plotter2D.py", line 358, in setSlicer
  File "sas\qtgui\Plotting\Slicers\AnnulusSlicer.py", line 50, in __init__
  File "sas\qtgui\Plotting\Slicers\AnnulusSlicer.py", line 109, in _post_data
  File "sas\sascalc\dataloader\manipulations.py", line 783, in __call__
UnboundLocalError: local variable 'err_data' referenced before assignment

Reply:
of course it should. Doh.
Thanks.